modules/data.land/inst/Pulling_VSWC.R

```diff
@@ -0,0 +1,60 @@
+# Script pulling soil moisture data from target files to create a netCDF file at the end
+## vswc = mass_fraction_of_unfrozen_water_in_soil_moisture
+
+# loading necessary packages
+library(readr)
+library(dplyr)
+library(ncdf4)
+
+# set function inputs
+homedir <- "/projectnb/dietzelab/cfranci1"
```

Comment:
But we will want to pull these hard-coded paths right?  so they work on any machine